_dashboards/branding.md

```diff
@@ -0,0 +1,89 @@
+---
+layout: default
+title: Custom branding
+nav_order: 200
+---
+
+# Dashboards custom branding
+Introduced 1.2
+{: .label .label-purple }
+
+By default, OpenSearch Dashboards uses the OpenSearch logo, but if you want to use custom branding elements such as the favicon or main Dashboards logo, you can do so by editing `opensearch_dashboards.yml` or by including a custom `opensearch_dashboards.yml` file when you start your OpenSearch cluster.
+
+For example, if you're using Docker to start your OpenSearch cluster, include the following lines in the `opensearch-dashboards` section of your `docker-compose.yml` file:
```

Comment:
NIT: should we specify that this will replace the Docker image's default `opensearch_dashboards.yml` with your local `opensearch_dashboards.yml`. Don't have to get much more detail than that but at least we mention it so people realize that they could potentially removing other default values (related to security plugin).

Reply:
Sure, that's useful information. I will add it in.